### DISCUSSION TITLE
Pyload Setup wt rc.pyload setup ->  [Errno 17] File exists: '/mod/pyload'

### DISCUSSION THREAD

**Discussion by Er4twXz** [Freetz-NG/freetz-ng#860] [Allgemein] [answered]
Wenn ich versuche pyload auf meiner Fritzbox 6660 zu initialisieren bekomme ich folgenden Fehler

Firmware: 252.07.57 rev107842 | Branding: avm
Freetz: ng-22611-d23cf858f

`root@fritz:~# rc.pyload setup
Traceback (most recent call last):
  File "/opt/pyLoad/pyLoadCore.py", line 43, in <module>
    from module import InitHomeDir
  File "/opt/pyLoad/module/InitHomeDir.py", line 73, in <module>
    makedirs(configdir, 0700) [Errno 17] File exists: '/mod/pyload'
  File "/usr/lib/python2.7/os.py", line 157, in makedirs
OSError: [Errno 17] File exists: '/mod/pyload'
`

**Comment by fda77** [comment]
Warum hast du die Datei erstellt? Lösche sie doch

**Reply by Er4twXz** [reply]
Gute Idee, auf die ich natürlich auch gekommen bin. Leider wird die Datei während der Ausführung des Skripts erstellt und verhindert dann den Abschluss des Skripts.

Ich denke ich weiß jetzt wo das Problem ist:

Es wird folgender Link angelegt : 
pyload -> /var/media/ftp/uStor01/pyload

Da ich keinen Stick in der Fritzbox stecken habe gibt es kein uStor01

**Comment by fda77** [comment]
Ohne Stick wirst du auch keinen Platz haben um das heruntergeladene zu speichern...

**Reply by Er4twXz** [reply]
Alles gut. Ich wollte pyload einfach nur mal ausprobieren und die Fehlermeldung war leider nicht sehr aussagekräftig.

Mit Stick wird es dann wohl funktionieren.

**Reply by fda77** [reply]
Zum testen kannst auch nach /tmp speichern, wenn der Ram voll ist wird die Box aber crashen/neustarten!

**Comment by Er4twXz** [comment, marked as answer]
Habe einen USB Stick angeschlossen den ich zuvor uStor01 benannt habe. Damit geht das Setup weiter.

Jetzt lande ich aber bei folgendem Problem:

https://github.com/Freetz-NG/freetz-ng/discussions/773
